docs/3.0rc/resources/upgrade-agents-to-workers.mdx

```diff
@@ -239,34 +252,31 @@ if __name__ == "__main__":
     )
 ```
 
-You can use `flow.from_source` to load your flow from the same location and `flow.serve` to 
+You can use `flow.from_source` to load your flow from the same location and `flow.deploy` to 
 create a deployment:
 
-```python
+```python example.py
```

Comment:
Maybe name the example files with different names to denote what's happening in them. Not a strongly held opinion.